src/Uno.UWP/UI/Core/CoreDispatcher.Skia.cs

```diff
@@ -6,17 +6,29 @@
 
 namespace Windows.UI.Core
 {
-    public sealed partial class CoreDispatcher
+	public sealed partial class CoreDispatcher
 	{
 		/// <summary>
 		/// Provide a action that will delegate the dispach of CoreDispatcher work
 		/// </summary>
 		[EditorBrowsable(EditorBrowsableState.Never)]
 		public static Action<Action> DispatchOverride;
 
-		public static bool HasThreadAccessOverride { get; set; } = true;
-		 
-		private bool GetHasThreadAccess() => HasThreadAccessOverride;
+		/// <summary>
+		/// Provide a action that will delegate the dispach of CoreDispatcher work
+		/// </summary>
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public static Func<bool> HasThreadAccessOverride;
```

Comment:
I would name this something else - it's not really an 'override' if it's obligatory to be set. `HasThreadAccessCheck`?

Reply:
I would agree, but same applies to the `DispatchOverride` ... I'll change both in another commit